Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -1,106 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{3E6328DD-E284-4F23-B55A-472BB7EA2C25}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Snowflake.Data</RootNamespace>
-    <AssemblyName>Snowflake.Data</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>

```

Comment:
Ok. this means both .Netcore 2.0 and .Net Framework 4.6 are supported?

Reply:
Right. Note this is the project file for .net driver itself. So I used the netstandard2.0, which did not have a runtime. (But should be good for compilation) Customer (or application) needs to use netcoreapp2.0 as target framework since .net core will have a runtime. Note the tests binaries are using .netcoreapp2.0 since ultimately, for tests to be run, a runtime is required.